includes/im_user_homepage.inc

```diff
@@ -59,30 +62,31 @@ function _im_user_login_validate(){
           //Maps stores with - User Profile.
           $user_stores = _get_user_stores($profile);
           watchdog('im_user', 'user_stores %user_stores', array('%user_stores'=>var_export($user_stores, TRUE)), WATCHDOG_NOTICE, $link = NULL);
-          if(sizeof($user_stores) > 0) {
+          if (sizeof($user_stores) > 0) {
             map_stores_profile($user_stores, $account);
           }
            //Assign Users email to profile.
-          if(isset($profile_name['mail']) && $profile_name['mail']) {
+          if (isset($profile_name['mail']) && $profile_name['mail']) {
             $account->mail = trim($profile_name['mail']);
             user_save($account);
            }
           //Assign Users full name to profile.
-          if(isset($profile_name['givenName']) && $profile_name['sn']) {
+          if (isset($profile_name['givenName']) && $profile_name['sn']) {
             $account->field_full_name[LANGUAGE_NONE][0]['value'] = trim($profile_name['givenName']) . ' ' . trim($profile_name['sn']);
             user_save($account);
           }
           //Assign DMD role to User Object.

```

Comment:
DMD Role is a User role defined by the Business specification mostly we are using the short format for our reference.